qa-ibd.planx-pla.net/portal/gitops.json

```diff
@@ -424,6 +430,37 @@
       "dropdowns": {}
     }
   ],
+  "studyViewerConfig": [
+    {
+      "dataType": "project",
+      "title": "Projects",
+      "titleField": "title",
+      "rowAccessor": "name",
```

Comment:
the `rowAccessor` needs to be a unique ID, so the name won't work. Let's use the `project_id`